### ISSUE TITLE
[TF2] The Third Degree balance suggestion

### ISSUE THREAD

**Issue by luki1412** [ValveSoftware/Source-1-Games#4613]
This weapon is better than stock because it has no downside but has an extra upside.
Maybe no random crits or -10% swinging speed to balance it or less damage on players that arent actively being healed.

**Comment by mtxfellen** [comment]
This weapon already sees very little use in play, as its upside still makes it inferior to any flamethrower. Nerfing it on grounds of technically being better than stock without argument (despite superior options existing) would be putting the cart before the horse, given that nerfing it at all would be unjustified.

**Comment by Morty-Mcfly** [comment]
> This weapon is better than stock because it has no downside but has an extra upside. Maybe no random crits or -10% swinging speed to balance it or less damage on players that aren't actively being healed.

good luck with that. I think that at this point, any balance changes you wish for should be made on your own via vscript

**Comment by luki1412** [comment]
> This weapon already sees very little use in play, as its upside still makes it inferior to any flamethrower. Nerfing it on grounds of technically being better than stock without argument (despite superior options existing) would be putting the cart before the horse, given that nerfing it at all would be unjustified.

Balance and how fun the weapon is to use or how often it is used, are not the same thing. Stop confusing different concepts. Weapons should be fun to use and balanced. Not easy to achieve but that's the objective. This weapon is boring AND unbalanced.

**Comment by mtxfellen** [comment]
> This weapon is boring AND unbalanced.

I agree wholeheartedly, and nerfing it would help this how, exactly?

**Comment by luki1412** [comment]
> > This weapon is boring AND unbalanced.
> 
> I agree wholeheartedly, and nerfing it would help this how, exactly?

It would make it boring and balanced, as my title would suggest.

Now that think about redesigning the weapon, heres an idea:

+ deals damage to enemies in a small radius around the target, on hit
- decreased damage by 40%
